tests/e2e_test/test_bench.py

```diff
@@ -99,27 +100,43 @@ async def test_simple_benchmark(setup_ray_env, model):
                                                  launch_ray_cluster=False, port=base_port+i, model=model)
         subprocess.run(launch_command, shell=True, check=True)
 
-    await asyncio.sleep(60)
+    await asyncio.sleep(30)
 
-    async def run_bench_command(command):
-        process = await asyncio.create_subprocess_shell(command)
-        await process.wait()
-        assert process.returncode == 0
+    def run_bench_command(command):
```

Comment:
for what?

Reply:
I don't know why asyncio.create_subprocess_shell doesn't return the process's error exit code
Just try a different way to start it.